postgres-appliance/bootstrap/maybe_pg_upgrade.py

```diff
@@ -32,6 +32,7 @@ def perform_pitr(postgresql, cluster_version, bin_version, config):
     config[config['method']]['command'] = 'true'
     try:
         if bin_version == cluster_version:
+            subprocess.Popen(["chmod 0700 /home/postgres/pgdata/pgroot/data"], shell=True)
```

Comment:
```suggestion
            os.chmod('/home/postgres/pgdata/pgroot/data', 0o700)
```

if the `os` module is imported at the top :)

Reply:
Thanks, I wasn't aware of this :+1: